SPECS/xenserver-status-report.spec

```diff
@@ -53,15 +91,78 @@ ln %{buildroot}/%{_mandir}/man1/%{bin0_name}.1 \
 %doc %{_mandir}/man1/%{bin0_name}.1.gz
 
 %changelog
-* Mon Aug 30 2021 Lin Liu <lin.liu@citrix.com> - 1.2.10-1
-- CP-36092: Backport winbind to Yangtze
```

Comment:
Seems strange that this changelog entry has been removed but I guess the spec file is now identical as the one from the imported package so it's ok

Reply:
I think they took the package from 8.3 and replaced the 8.2 one with it, loosing the 8.2-specific changelog entry in the process.